### PR TITLE
Fixes install instructions in README

### PR DESCRIPTION
**Pull request by rafariossaa**
**Description of the change**

Fix install instructions documentation

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [-] Version bumped in `package.json` and `package-lock.json` according to [semver](http://semver.org/).
- [-] New features are documented in the README.md
- [-] New features contain a new test at the `tests` folder
- [-] All tests pass running `npm test`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Execute the following commands to install the tool:
 
 ```console
 git clone https://github.com/bitnami-labs/readme-generator-for-helm
-npm install -g readme-generator-for-helm
+sudo npm install -g ./readme-generator-for-helm
```
[the remark this answers is not in the document]
Comment (rafariossaa) [reply README.md:47]: you should not tinker with the permissions of those directories.
If you use `-g` you have to do it with `sudo`

[the remark this answers is not in the document]
Comment (miguelaeh) [reply README.md:47]: The `-g` option install the packages into the configured prefix. Check [this](https://docs.npmjs.com/cli/v8/configuring-npm/folders#prefix-configuration).

Taking that into account, when installing node, if it has been installed into a "global" system path, i.e. for all the users, for example, it is typical to install it under `/usr/local` it will require `sudo`.
If the system has more users, it would be a good practise that each user will have its own node installation, they could want to use different versions, etc. In that case, let's say you install it under `${HOME}/bin`. The prefix in that case won't need root permissions.

So, in short, I think this shouldn't be added by default. Each user should know how its system is configured, and use `sudo` if the certain case requires it. But setting it by default will be telling to the users to use `sudo` always, which depending on your node installation path could or could not be necessary.

[the remark this answers is not in the document]
Comment (recena) [reply README.md:47]: @rafariossaa Is there any consensus?

[the remark this answers is not in the document]
Comment (rafariossaa) [reply README.md]: We agreed in changing the instructions and adding a small comment for the user.
Depending on how the user has installed node he might need to adjust the `PATH`.

**Review by miguelaeh — APPROVED** [review]
LGTM
